catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/model/Search.java

```diff
@@ -48,6 +51,17 @@
  */
 public class Search {
 
+    public static final String REASON_UNSUPPORTED_QUERY = "Query was invalid";
```

Comment:
Any reason those constants are `public`? If it is for unit testing purposes, we should make them `private` and hard-code the same values in the test class.